internal/provider/virtual_clusters_data_source_test.go

```diff
@@ -0,0 +1,72 @@
+package provider
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestAccVirtualClustersDataSource(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccVirtualClustersDataSource_default(),
+				Check:  testAccVCsDataSourceCheckServerless("tivo_serverless"),
+			},
+			{
+				Config: testAccVirtualClustersDataSource_default(),
+				Check:  testAccVCsDataSourceCheckBYOC("default"),
+			},
+		},
+	})
+}
+
+func testAccVirtualClustersDataSource_default() string {
+	return providerConfig + `
+data "warpstream_virtual_clusters" "test" {
+}`
+}
+
+func testAccVCsDataSourceCheckServerless(serverlessVCName string) resource.TestCheckFunc {
+	return resource.ComposeAggregateTestCheckFunc(
+		// resource.TestCheckResourceAttr("data.warpstream_virtual_clusters.test", "virtual_clusters.#", "5"),
```

Comment:
TODO: make sure tf acceptance tests have their own warpstream tenant and no one else is using it.

Reply:
What do you propose exactly? Is it more than changing the name of the tenant currently used?

Reply:
It was another sleepy brain comment. I was wondering if Proby was doing stuff in the tf test tenant and changing the state from what the test suite expects. Now I think it's more likely it's the test suite itself running in parallel and temporarily changing the state! That would explain why tests that pass locally fail on unexpected tenant state in CI.

Reply:
Proby is using its dedicated separate account, so it should not be the case indeed.